docs/source/nbconvert_library.ipynb

```diff
@@ -511,27 +511,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
+    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
```

Comment:
Had the same question this morning, the old ipython/nbconvert repo seem to have been deleted. 

Remember we developed nbconvert as a separate package, merged it into IPython for a few years, and split it into Jupyter. so IPython/notebook != Jupyter notebook, and I actually had a sociologue that ask me were was the old repo, and I have no clue. @fperez ? @ellisonbg ?